certified-connectors/Weavo Liquid Loom/readme.md

```diff
@@ -0,0 +1,28 @@
+# Weavo Liquid Loom Connector for Power Automate
```

Comment:
Can you please update readme.md file with all required section mentioned in the below link?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
done